src/Entities/Misc/RailedMoveBlock.cs

```diff
@@ -283,11 +285,11 @@ public RailedMoveBlock(Vector2 position, int width, int height, Vector2 node, St
                 float newSpeed = 0f;
 
                 icon = idleIcon;
-                if (hasTopButtons && GetPlayerOnTop() != null && Input.MoveY.Value != 0) {
+                if ((topPressed || bottomPressed) && Input.MoveY.Value != 0) {
                     newSpeed = moveSpeed * Input.MoveY.Value * (dir.Y > 0f ? 1f : -1f);
                     newFillColor = MoveBgFill;
                     icon = Input.MoveY.Value == 1 ? DownIcon : UpIcon;
-                } else if (hasSideButtons && GetPlayerClimbing() != null && Input.MoveX.Value != 0) {
+                } else if ((leftPressed || rightPressed) && Input.MoveX.Value != 0) {
```

Comment:
As far as I know, a button being pressed doesn't correlate directly to moving the block, they can be visually pressed in but the block won't be steered unless the player is grabbing it.

Reply:
fixed in https://github.com/CommunalHelper/CommunalHelper/commit/91966506be7d943992f77db41c035494aca6c892